msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -179,8 +180,7 @@ All users in a meeting receive the notifications sent through in-meeting notific
 > [!NOTE]
```

Comment:
Can we add some info here about targeted notifications. Users might look at this doc and miss that targeted notifications are at the bottom.